### ISSUE TITLE
Loss of language support in the latest nightly version

### ISSUE THREAD

**Issue by eduardoaddad** [AdguardTeam/AdGuardVPNForWindows#306]
Loss of language support in the latest nightly version and bug 

Version: 2.0 nightly 1

![image](https://user-images.githubusercontent.com/16819380/185771401-cd972065-0c2f-4423-9c2a-bfcbfbb33ae4.png)
![image](https://user-images.githubusercontent.com/16819380/185771414-17cf196c-e488-4a33-ae3f-0f625b3b3478.png)
![image](https://user-images.githubusercontent.com/16819380/185771421-72d56d2d-277a-440d-bbb2-00db464eac73.png)
![eror lang 2](https://user-images.githubusercontent.com/16819380/185771668-b626a61c-ffee-4b61-8c73-3f948ced0efb.png)

```
INFO, AdGuardVpnSvc, 20, 20.08.2022 22:18:41.711, No language is set, use fallback from the file C:\ProgramData\AdguardVPN\locale.dat
ERROR, AdGuardVpn, 1, 02.07.2022 0:12:14.483, Cannot load assembly C:\Program Files (x86)\AdGuardVpn\langs\AdGuardVpn.UI.resources.pt-BR.dll
INFO, AdGuardVpn, 1, 02.07.2022 0:12:14.485, Fallback from pt-BR to pt
INFO, AdGuardVpn, 1, 02.07.2022 0:12:14.539, Assembly C:\Program Files (x86)\AdGuardVpn\langs\AdGuardVpn.UI.resources.pt.dll has been loaded
```

**Comment by mozartinio** [comment]
@eduardoaddad Hi! We know about it and will add another languages later. VPN 2.0 version is more about new changes/features and the main focus is on productivity!
Thank you for your attention!

**Comment by gork7777** [comment]
Looks like fixed in v2.0.664.0
